src/core/server/opensearch/version_check/ensure_opensearch_version.test.ts

```diff
@@ -356,6 +366,154 @@ describe('pollOpenSearchNodesVersion', () => {
       });
   });
 
+  it('returns compatibility results and isCompatible=true with filters and multiple cluster ids', (done) => {
```

Comment:
nit: It would be helpful to add to the test title when you expect these results. It isnt clear to me when i should expect to see `isCompatible=true`..etc